ros1_ign_image/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ros1_ign_image)
+
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra)
+endif()
+
+find_package(catkin REQUIRED COMPONENTS
+  image_transport
+  ros1_ign_bridge
+  roscpp
+  sensor_msgs)
+
+find_package(ignition-msgs4 QUIET REQUIRED)
+set(IGN_MSGS_VER ${ignition-msgs4_VERSION_MAJOR})
+
+find_package(ignition-transport7 QUIET REQUIRED)
+set(IGN_TRANSPORT_VER ${ignition-transport7_VERSION_MAJOR})
+
+catkin_package()
+
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+)
+
+set(executable
+  image_bridge
+)
+
+add_executable(${executable}
+  src/image_bridge.cpp
+)
+target_link_libraries(${executable}
+  ${catkin_LIBRARIES}
+  ignition-msgs${IGN_MSGS_VER}::core
+  ignition-transport${IGN_TRANSPORT_VER}::core
+)
+install(TARGETS ${executable}
+        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+# Tests
+find_package(rostest REQUIRED)
+
+set(test_publishers
+  ign_publisher
```

Comment:
I'm getting a few errors about target name conflicts with the ones declared in `ros1_ign_bridge` when building the workspace using `catkin_make`

I just had to change the target name. This worked for me:

```
diff --git a/ros1_ign_image/CMakeLists.txt b/ros1_ign_image/CMakeLists.txt
index 03d926c..e80fbd2 100644
--- a/ros1_ign_image/CMakeLists.txt
+++ b/ros1_ign_image/CMakeLists.txt
@@ -56,10 +56,10 @@ set(test_subscribers
 )
 
 foreach(test_publisher ${test_publishers})
-  add_executable(${test_publisher}
+  add_executable(${test_publisher}_image
     test/publishers/${test_publisher}.cpp
   )
-  target_link_libraries(${test_publisher}
+  target_link_libraries(${test_publisher}_image
     ${catkin_LIBRARIES}
     ignition-msgs${IGN_MSGS_VER}::core
     ignition-transport${IGN_TRANSPORT_VER}::core
@@ -69,10 +69,10 @@ foreach(test_publisher ${test_publishers})
 endforeach(test_publisher)
 
 foreach(test_subscriber ${test_subscribers})
-  add_rostest_gtest(test_${test_subscriber}
+  add_rostest_gtest(test_${test_subscriber}_image
     test/${test_subscriber}.test
     test/subscribers/${test_subscriber}.cpp)
-  target_link_libraries(test_${test_subscriber}
+  target_link_libraries(test_${test_subscriber}_image
     ${catkin_LIBRARIES}
     ignition-msgs${IGN_MSGS_VER}::core
     ignition-transport${IGN_TRANSPORT_VER}::core
diff --git a/ros1_ign_image/test/launch/test_ros1_subscriber.launch b/ros1_ign_image/test/launch/test_ros1_subscriber.launch
index 963baec..b280f65 100644
--- a/ros1_ign_image/test/launch/test_ros1_subscriber.launch
+++ b/ros1_ign_image/test/launch/test_ros1_subscriber.launch
@@ -7,6 +7,6 @@
   />
 
   <!-- Launch the Ignition Transport publisher -->
-  <node name="ign_publisher" pkg="ros1_ign_image" type="ign_publisher" />
+  <node name="ign_publisher" pkg="ros1_ign_image" type="ign_publisher_image" />
 
 </launch>
diff --git a/ros1_ign_image/test/ros1_subscriber.test b/ros1_ign_image/test/ros1_subscriber.test
index 6c43831..a32d713 100644
--- a/ros1_ign_image/test/ros1_subscriber.test
+++ b/ros1_ign_image/test/ros1_subscriber.test
@@ -4,6 +4,6 @@
   <include file="$(find ros1_ign_image)/test/launch/test_ros1_subscriber.launch">
   </include>
 
-  <test test-name="ign_ros1" pkg="ros1_ign_image" type="test_ros1_subscriber" time-limit="60.0" />
+  <test test-name="ign_ros1" pkg="ros1_ign_image" type="test_ros1_subscriber_image" time-limit="60.0" />
 
 </launch>
```

Reply:
Interesting, colcon must keep tests isolated even with `merge-install`. Added your changes: 34ba902f35913ae58b2a4422e29ab45b30e3bbb7